test/src/main/java/org/apache/accumulo/test/functional/SummaryIT.java

```diff
@@ -0,0 +1,759 @@
+    Assert.assertEquals(0, summaries.size());
+
+    LongSummaryStatistics stats = getTimestampStats(table, c);
+
+    summaries = c.tableOperations().getSummaries(table).flush(true).retrieve();
+    checkSummaries(summaries, sc1, 1, 0, 0, "total", 100_000l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    Mutation m = new Mutation(String.format("r%09x", 999));
+    m.put("f1", "q1", "999-0");
+    m.putDelete("f1", "q2");
+    bw.addMutation(m);
+    bw.flush();
+
+    c.tableOperations().flush(table, null, null, true);
+
+    stats = getTimestampStats(table, c);
+
+    summaries = c.tableOperations().getSummaries(table).retrieve();
+
+    checkSummaries(summaries, sc1, 2, 0, 0, "total", 100_002l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 1l);
+
+    bw.close();
+
+    c.tableOperations().compact(table, new CompactionConfig().setWait(true));
+
+    summaries = c.tableOperations().getSummaries(table).retrieve();
+    checkSummaries(summaries, sc1, 1, 0, 0, "total", 100_000l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    // split tablet into two
+    String sp1 = String.format("r%09x", 50_000);
+    addSplits(table, c, sp1);
+
+    summaries = c.tableOperations().getSummaries(table).retrieve();
+
+    checkSummaries(summaries, sc1, 1, 0, 0, "total", 100_000l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    // compact 2nd tablet
+    c.tableOperations().compact(table, new CompactionConfig().setStartRow(new Text(sp1)).setWait(true));
+
+    summaries = c.tableOperations().getSummaries(table).retrieve();
+    checkSummaries(summaries, sc1, 2, 0, 1, "total", 113_999l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    // get summaries for first tablet
+    stats = getTimestampStats(table, c, sp1, null);
+    summaries = c.tableOperations().getSummaries(table).startRow(sp1).retrieve();
+    checkSummaries(summaries, sc1, 1, 0, 0, "total", 49_999l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    // compact all tablets and regenerate all summaries
+    c.tableOperations().compact(table, new CompactionConfig());
+
+    summaries = c.tableOperations().getSummaries(table).retrieve();
+    stats = getTimestampStats(table, c);
+    checkSummaries(summaries, sc1, 2, 0, 0, "total", 100_000l, "minStamp", stats.getMin(), "maxStamp", stats.getMax(), "deletes", 0l);
+
+    summaries = c.tableOperations().getSummaries(table).startRow(String.format("r%09x", 75_000)).endRow(String.format("r%09x", 80_000)).retrieve();
+    Summary summary = Iterables.getOnlyElement(summaries);
+    Assert.assertEquals(1, summary.getFileStatistics().getTotal());
+    Assert.assertEquals(1, summary.getFileStatistics().getExtra());
+    long total = summary.getStatistics().get("total");
+    Assert.assertTrue(total > 0 && total <= 10_000);
```

Comment:
Nit: list what the total was in the message argument for `assertTrue(String, boolean)`.